File: src/main/java/com/pCarpet/dto/UserDTO.java
package com.pCarpet.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDTO {

	private Integer idUser;

	private String username;
	
	private String password;
	
	private String nome;
	
	private String cognome;
	
	private String email;
	
	private String lingua;
	
	
	
}

File: Front-end/src/app/dto/UserDTO.ts
export class UserDTO
{
    idUser: number;
    username: String;
    password: String;
    nome: String;
    cognome: String;
    email: String;
    lingua: String;
    rank: number;

    constructor(idUser: number, username: String, password: String, nome: String, cognome: String, email: String, lingua: String, rank: number)
    {
        this.idUser = idUser;
        this.username = username;
        this.password = password;
        this.nome = nome;
        this.cognome = cognome;
        this.email = email;
        this.lingua = lingua;
        this.rank = rank;

    }

}